libafl_cc/src/clang.rs

```diff
@@ -72,7 +72,7 @@ pub struct ClangWrapper {
 
 #[allow(clippy::match_same_arms)] // for the linking = false wip for "shared"
 impl CompilerWrapper for ClangWrapper {
-    fn from_args<S>(&mut self, args: &[S]) -> Result<&'_ mut Self, Error>
```

Comment:
This changes the API but else rust complains that `from` should take a `self`, @andreafioraldi what do you think